Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM registry.access.redhat.com/ubi8/python-38
+
+WORKDIR /archspec
+COPY . .
+
+USER root
+
+RUN dnf update -y && dnf install git python38 python3-pip -y
+RUN pip3 install poetry tox-gh-actions coverage
+RUN pip3 install archspec
+RUN dnf clean all
+
+RUN useradd archer
+USER archer
```

Comment:
Who's `archer`?

Reply:
no one really, just a name that came to my head when assigning a non-root user to run archspec

Reply:
I am open to change the running user to what ever name seems more appropriate.
`ArchSpec-runner` ?

Reply:
How about just archspec?

Reply:
afd85e7